.meta/webpack.base.config.js

```diff
@@ -111,11 +110,5 @@ module.exports = {
   ],
   plugins: [
     new ExtractTextPlugin("foundation.css")
-  ],
-  sassLoader: {
-    includePaths: [
-      path.resolve(node_modules_path, "foundation-sites/scss"),
```

Comment:
I can't say, that this approach will fit well. It would be better to use foundation code like from current path, settings file will not differ from standard one, so, users can just replace it with our file.